ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/response/JsonRpcError.java

```diff
@@ -77,32 +77,32 @@
   PARENT_BLOCK_NOT_FOUND(-32000, "Parent block not found"),
 
   // Permissioning/Account allowlist errors
-  ACCOUNT_WHITELIST_NOT_ENABLED(-32000, "Account allowlisting has not been enabled"),
-  ACCOUNT_WHITELIST_EMPTY_ENTRY(-32000, "Request contains an empty list of accounts"),
-  ACCOUNT_WHITELIST_INVALID_ENTRY(-32000, "Request contains an invalid account"),
-  ACCOUNT_WHITELIST_DUPLICATED_ENTRY(-32000, "Request contains duplicate accounts"),
-  ACCOUNT_WHITELIST_EXISTING_ENTRY(-32000, "Cannot add an existing account to allowlist"),
-  ACCOUNT_WHITELIST_ABSENT_ENTRY(-32000, "Cannot remove an absent account from allowlist"),
+  ACCOUNT_ALLOWLIST_NOT_ENABLED(-32000, "Account allowlisting has not been enabled"),
+  ACCOUNT_ALLOWLIST_EMPTY_ENTRY(-32000, "Request contains an empty list of accounts"),
+  ACCOUNT_ALLOWLIST_INVALID_ENTRY(-32000, "Request contains an invalid account"),
+  ACCOUNT_ALLOWLIST_DUPLICATED_ENTRY(-32000, "Request contains duplicate accounts"),
+  ACCOUNT_ALLOWLIST_EXISTING_ENTRY(-32000, "Cannot add an existing account to allowlist"),
+  ACCOUNT_ALLOWLIST_ABSENT_ENTRY(-32000, "Cannot remove an absent account from allowlist"),
 
   // Permissioning/Node allowlist errors
-  NODE_WHITELIST_NOT_ENABLED(-32000, "Node allowlisting has not been enabled"),
-  NODE_WHITELIST_EMPTY_ENTRY(-32000, "Request contains an empty list of nodes"),
-  NODE_WHITELIST_INVALID_ENTRY(-32000, "Request contains an invalid node"),
-  NODE_WHITELIST_DUPLICATED_ENTRY(-32000, "Request contains duplicate nodes"),
-  NODE_WHITELIST_EXISTING_ENTRY(-32000, "Cannot add an existing node to allowlist"),
-  NODE_WHITELIST_MISSING_ENTRY(-32000, "Cannot remove an absent node from allowlist"),
-  NODE_WHITELIST_FIXED_NODE_CANNOT_BE_REMOVED(
+  NODE_ALLOWLIST_NOT_ENABLED(-32000, "Node allowlisting has not been enabled"),
```

Comment:
"allowlisting" or "allowlist" ?

Reply:
I do not have a strong opinion on this

Reply:
ok we're deciding that allowlist is a noun not a verb. but changes will come in a new PR